swagger.yaml

```diff
@@ -412,8 +509,65 @@ definitions:
         description: |
           The name of the filter, as it is definined in Skipper
       args:
+        description: The arguments for the filter, can be string or float64
+        type: array
+        items:
+          type: string
+  Host:
+    required:
+      - id
+      - name
+    properties:
+      id:
+        type: string
+        format: int32
+      name:
+        type: string
+    example: |
+      {
+        "id": "1",
+        "name": "zalando.de"
+      }
+  PathIn:
+    required:
+      - uri
+      - host_ids
+    properties:
+      uri:
+        type: string
+      host_ids:
+        type: array
+        items:
+          type: string
+    example: |
+      {
+        "uri": "/cart",
+        "host_ids": ["1", "2"]
+      }
+  PathOut:
+    required:
+      - id
+      - team
```

Comment:
The Route has  `owned_by_team` and `created_by`, why does this have only one field and named differently?